src/services/platform.service.ts

```
@@ -144,6 +144,15 @@ const updatePlatform = async (
  * @returns {Promise<HydratedDocument<IPlatform>>}
  */
 const deletePlatform = async (platform: HydratedDocument<IPlatform>): Promise<HydratedDocument<IPlatform>> => {
+  switch (platform.name) {
+    case PlatformNames.Discourse: {
+      if (platform.metadata?.scheduleId) {
+        await discourseService.coreService.deleteDiscourseSchedule(platform.metadata.scheduleId);
+      }
+    }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)